### PR TITLE
WT-4348 In directio test, create all tables before spawning threads.

### PR DESCRIPTION
**Pull request by ddanderson**

### CODE REVIEW

**Comment by ddanderson** [comment]
@sueloverso, can you please review?

File: test/csuite/random_directio/main.c [hunk 1]
```diff
@@ -593,6 +588,37 @@ thread_run(void *arg)
 	/* NOTREACHED */
 }
 
+/*
+ * create_db --
+ *	Creates the database and tables so they are fully ready to
```
Comment (sueloverso) [inline test/csuite/random_directio/main.c]: should be "ready to be"

File: test/csuite/random_directio/main.c [hunk 2]
```diff
@@ -593,6 +588,37 @@ thread_run(void *arg)
...
+	 * can be opened.
+	 */
+	testutil_check(session->checkpoint(session, NULL));
+	testutil_check(session->close(session, NULL));
+	testutil_check(conn->close(conn, NULL));
+}
+
 /*
  * fill_db --
  *	The child process creates the database and table, and then creates
```
Comment (sueloverso) [inline test/csuite/random_directio/main.c]: This comment for `fill_db` seems out of date now.

File: test/csuite/random_directio/main.c [hunk 3]
```diff
@@ -601,7 +627,7 @@ thread_run(void *arg)
 static void fill_db(uint32_t, uint32_t, const char *, uint32_t)
     WT_GCC_FUNC_DECL_ATTRIBUTE((noreturn));
 static void
-    fill_db(uint32_t nth, uint32_t datasize, const char *method, uint32_t flags)
+fill_db(uint32_t nth, uint32_t datasize, const char *method, uint32_t flags)
 {
 	WT_CONNECTION *conn;
 	WT_SESSION *session;
```
Comment (sueloverso) [inline test/csuite/random_directio/main.c]: Should there be `create` lines removed from here now? They're probably not harmful, but possibly confusing.

**Pull request merged**